governance/program/src/state/proposal.rs

```diff
@@ -200,6 +199,9 @@ pub struct ProposalV2 {
 
     /// Link to proposal's description
     pub description_link: String,
+
+    /// The total weight of Veto votes
+    pub veto_vote_weight: u64,
```

Comment:
The reason I moved it to the end of the struct and replaced with `reserved1: u8` was because it was unused  `Option<u64>` before. It means it was always serialised as `0:u8` and I couldn't change it to `u64` because it wouldn't be able to deserialise existing accounts where it would always wanted to use 8 consecutive bytes.


The only risk with this was if the actual serialised size of  `Proposal` account could ever be shrank because I don't think we zero the reaming empty account space, so we could end up  with `veto_vote_weight` being read from some leftovers. But the shrinking can never happen in the current design